packages/imperative/src/config/__tests__/ProfileInfo.TeamConfig.unit.test.ts

```diff
@@ -1432,6 +1432,8 @@ describe("TeamConfig ProfileInfo tests", () => {
                 const blockMocks = getBlockMocks();
                 const profInfo = createNewProfInfo(teamProjDir);
                 await profInfo.readProfilesFromDisk({ homeDir: teamHomeProjDir });
+                jest.spyOn(Map.prototype, "has").mockReturnValue(true);
+                jest.spyOn(lodash, "isEqual").mockReturnValue(true);
```

Comment:
Thi is an interesting way to get to the desired code. 😋 
Glad that we have a `jest.restoreAllMocks()` call in the `afterEach()`

LGTM! :yum: